tests/test_queue.py

```diff
@@ -1,43 +1,43 @@
 import pytest
-from bayes_opt.bayesian_optimization import Queue
+from queue import Queue, Empty
 
 
 def test_add():
     queue = Queue()
 
-    assert len(queue) == 0
+    assert queue.qsize() == 0
     assert queue.empty
```

Comment:
I think this line should be updated to `queue.empty()`

Reply:
agree :)